### DISCUSSION TITLE
Diskusija na pregled komita [afbefbb]

### DISCUSSION THREAD

**Discussion by VuceticBranislav** [HECalculator/HECalculator#11] [General]
Diskusija vezana za komit: https://github.com/HECalculator/HECalculator/commit/afbefbbd70d694f251420134f1c966b13e0f8000

Ovde cu da pokusam da dam komentare na postojeci komit koji nije deo pull requesta.
Kao primer kako moze veca diskusija da se izvede u slucaju potrebe. Napravicu i Issue i project task vezano za ovo kako bi imali i neko pracenje napretka. Ubacicu linkove na sve sto ee moze linkovati :).

Issue povezan sa ovom temom je #9. Cekiracemo kako nesto uradimo da se vidi progres.

Komentar sa https://github.com/HECalculator/HECalculator/commit/afbefbbd70d694f251420134f1c966b13e0f8000#commitcomment-47085773:

> Vidim par stvari koje bi se mogle popraviti za sad:
>- Prebaciti klasu HECalcProcedural iz roota projekta u paket "/src/liftingLug".
>- Dodati u isti paket TextIO. Trenutno nam fali taj fajl. Mozda samo nije komitovan.
>- Pokrenuti proracun preko glavne klase "src/liftingLug/Project.java". U mainu pozvati metod `HECalcProcedural.main(args);`.

Poenta ove zadnje tacke jeste da "Project" bude glavna klasa. Trenutno imamo proceduralno resenje u odvojenom fajlu. Samo bi pozvali main funkciju iz tog fajla i dobili program koji radi. 
Kada dodje OOP resenje na red nastavili bi u "Project" klasi a poziv ka proceduralnoj verziji `HECalcProcedural.main(args);`  bi obrisali.

Pored ovoga ima jos par manjih komentara na komit.
https://github.com/HECalculator/HECalculator/commit/afbefbbd70d694f251420134f1c966b13e0f8000#r47085758
https://github.com/HECalculator/HECalculator/commit/afbefbbd70d694f251420134f1c966b13e0f8000#r47085767
https://github.com/HECalculator/HECalculator/commit/afbefbbd70d694f251420134f1c966b13e0f8000#r47085803

Ostalo mi deluje ok za ovu fazu.

**Comment by ivantot** [comment]
Kako izbaciti procedural fajl iz roota sad kad je u src folderu?

**Comment by VuceticBranislav** [comment]
pa kontam samo obrises pre komitovanja.. komentarisacu na pull rekvestu uz ostalo.
